.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
```

Comment:
Would it be possible to move this to the `pyproject.toml` file since we have it already?

Reply:
Also, I didn't see anything here for black compatibility -- did you mean to include:
```
profile = "black"
```

Reply:
oops yes put it here cause that was created by isort-config
Regarding black compatibility, I had that in the `.pre-commit-config.yaml`. But I like the idea of having it in `pyproject.toml` instead, so did that now

Reply:
Regarding order: I think it shouldn't matter. With the profiling the two should be compatible. If you don't do the profiling and run isort first, it is not working as black and isort will then undo each others changes and precommit doesnt like that

Reply:
> Regarding black compatibility, I had that in the .pre-commit-config.yaml.

Ah oops sorry I didn't spot it there, my bad

> Regarding order: I think it shouldn't matter. With the profiling the two should be compatible. If you don't do the profiling and run isort first, it is not working as black and isort will then undo each others changes and precommit doesnt like that

Sounds good!